pkg/client/attach.go

```diff
@@ -270,7 +270,7 @@ func (c *ConmonClient) redirectResponseToOutputStreams(
 				return nil
 			}
 		}
-		if er == io.EOF || (cfg.ContainerStdin && !cfg.StopAfterStdinEOF) {
+		if er == io.EOF && cfg.ContainerStdin && !cfg.StopAfterStdinEOF {
```

Comment:
Hm, we have to test if this works with CRI-O, I know we had some issues with that in the past.

Reply:
The more I think about it, it probably should just be
```go
if er == io.EOF {
    return nil
}
```

Reply:
`cfg.ContainerStdin && !cfg.StopAfterStdinEOF` feels like a hack, is that related to https://github.com/containers/conmon-rs/pull/1655?

Reply:
Likely we need to detect the EOF and then drain the streams.